src/Microsoft.ML.TimeSeries/ExtensionsCatalog.cs

```diff
@@ -156,7 +156,7 @@ public static class TimeSeriesCatalog
         /// <param name="outputColumnName">Name of the column resulting from data processing of <paramref name="inputColumnName"/>.
         /// The column data is a vector of <see cref="System.Double"/>. The length of this vector varies depending on <paramref name="detectMode"/>.</param>
         /// <param name="inputColumnName">Name of column to process. The column data must be <see cref="System.Double"/>.</param>
-        /// <param name="threshold">The threshold to determine anomaly, score larger than the threshold is considered as anomaly. Must be in [0,1]. Default value is 0.3.</param>
+        /// <param name="threshold">The threshold to determine anomaly. Scores larger than the threshold are considered as anomalies. This value must fall between [0,1]. Default value is 0.3.</param>
```

Comment:
Comment from @gvashishtha on the closed PR:
"Definition of threshold refers to "score," which I don't see defined anywhere"

Can you please add a note in comments on how to explain score here? I can make the fix.

Reply:
The suggestion from msftbl looks good to me. There is a similar comment in the function above